### PR TITLE
refactor(operators): add BaseTaskOperator and migrate Luma as PoC

### PR DESCRIPTION
**Pull request by acedatacloud-dev**
## Summary

Twelve of the eighteen per-service operators (`luma`, `seedance`, `sora`, `veo`, `kling`, `hailuo`, `wan`, `flux`, `nanobanana`, `pika`, `pixverse`, `qrart`) are **byte-identical except for two strings** — the `/<service>/tasks` and `/<service>/<resource>` paths. Each one hand-rolls ~110 lines of axios + header + filter-unrolling boilerplate.

Add a generic `BaseTaskOperator` that encapsulates this once. Migrate **Luma** as a proof of concept; leave the other eleven untouched on this PR so the diff is easy to review.

## What's in `src/operators/baseTaskOperator.ts`

```ts
class BaseTaskOperator<TGenReq, TGenResp, TTaskResp, TTasksResp, TFilter> {
  constructor(paths: { tasksPath: string; generatePath: string });

  task(id, { token })          // -> { action: 'retrieve', id }
  tasks(filter, { token })     // -> { action: 'retrieve_batch', ...filter }
  generate(data, { token })    // -> data
}
```

Headers (`accept`, `content-type`, `authorization`, `x-record-exempt` for the task endpoints; `application/x-ndjson` for generate) and `BASE_URL_API` are wired up internally — same as before.

The filter helper converts `camelCase` → `snake_case` and drops `undefined`, `null`, and `''`. That reproduces the previous hand-rolled semantics **exactly**:

| Original code (per-key) | New helper |
|---|---|
| `filter.ids ? { ids } : {}` (truthy check) | drops `undefined`, `null`, `''`; **keeps `[]`** because `Boolean([]) === true` |
| `filter.userId ? { user_id } : {}` (truthy) | drops `undefined`, `null`, `''` |
| `filter.limit !== undefined ? { limit } : {}` | drops `undefined` only — `0` is preserved |

## Migrated: `src/operators/luma.ts` (106 → 16 lines)

```ts
class LumaOperator extends BaseTaskOperator<
  ILumaGenerateRequest, ILumaGenerateResponse,
  ILumaTaskResponse, ILumaTasksResponse, ITaskListFilter
> {
  constructor() {
    super({ tasksPath: '/luma/tasks', generatePath: '/luma/videos' });
  }
}
export const lumaOperator = new LumaOperator();
```

### Equivalence test (9 cases, including edge cases)

```
{ userId: 'u1', createdAtMin: 1700000000, createdAtMax: 1700100000 }  ✓
{ userId: undefined, createdAtMin: 0 }            ← 0 preserved        ✓
{ ids: ['a','b'], applicationId: 'app1' }                              ✓
{ ids: [] }                ← kept (truthy)                             ✓
{ applicationId: '' }      ← dropped                                   ✓
{ applicationId: null }    ← dropped                                   ✓
{ limit: 10, offset: 0 }   ← 0 preserved                               ✓
{ limit: 0, offset: undefined }                                        ✓
{}                                                                     ✓

9 cases, 0 mismatches
```

### Build / lint

- `npm run build` (web surface) ✅ passes
- Lint failures shown in CI are pre-existing on `main` (`NotFound.vue`, `kling/capabilities.ts`, plus the auto-translate PR #608 fallout) — none touch files this PR modifies.

## Stats

| | |
|---|---|
| Lines removed (luma.ts) | 100 |
| Lines added (luma.ts) | 10 |
| Lines added (baseTaskOperator.ts) | 134 |
| **Net this PR** | **+44 lines** |
| **Per-service savings going forward** | **~95 lines / operator** |
| **Estimated savings if all 11 remaining services migrate** | **~1,000 lines** |

## Risk

Low. The base class produces functionally identical request bodies (proven by the equivalence test) and uses the same `BASE_URL_API`, headers, and HTTP verb. Behavior is preserved; only the location of the code changes.

## Why land this incrementally instead of migrating everything in one PR

1. Reviewable. A 1,200-line refactor PR is hard to merge cleanly when other Nexior PRs are in flight (Suno/Producer worktrees in particular touch their own operators).
2. Bisectable. If a regression is introduced for any one service, only that service's PR needs reverting — not the whole batch.
3. Each follow-up migration is mechanical and can be done in <15 minutes per operator (subclass declaration + exported instance). Future PRs can include 2-3 services at a time as those services are touched for other reasons.

The base class is intentionally generic over the request/response types so each subclass keeps its own typed contract — there is no loss of static safety from this refactor.

### CODE REVIEW

File: src/operators/luma.ts [hunk 1]
```diff
@@ -1,105 +1,15 @@
-import axios, { AxiosResponse } from 'axios';
 import { ILumaGenerateRequest, ILumaGenerateResponse, ILumaTaskResponse, ILumaTasksResponse } from '@/models';
```
Comment (Germey) [inline src/operators/luma.ts:1]: 就 luma 用了这个？？？？

**Review by Germey — CHANGES_REQUESTED** [review]
